pkgs/applications/gis/qgis/wrapped.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, lib, fetchurl, openssl, python3Packages, makeWrapper, symlinkJoin
+, qgis3-unwrapped
+}:
+with lib;
+let
+  # the python packages that should be included in the PYTHONPATH
+  pythonInputs = with python3Packages;
+    qgis3-unwrapped.pythonBuildInputs ++ [ chardet dateutil pyyaml pytz requests urllib3 ];
```

Comment:
Why should these few be included in the wrapper, but *not* in the build? Are they needed for specific plugins?

Reply:
Moved to build. The reference errors these resolved only showed up at runtime.
I would expect these are transitive dependencies for packages missing respective propagatedBuildInputs?

Reply:
They are not hard dependencies otherwise the builds of the dependents would fail. Either upstream has forgotten to list their full dependencies, or they require "extras" of certain packages.
https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies